app/views/games/new.html.erb

```diff
@@ -1,6 +1,41 @@
-<h1>Create A New Game</h1>
-<%= simple_form_for @game, url: games_path do |f| %>
-  <%= f.input :name %>
+<div class="booyah-box">
+
+  <div class="create-game">
+    <div class="row">
+        <div class="col-12">
+          <h1>Create Game</h1>
+        </div>
+      <div class="form-box row">
+          <%= simple_form_for @game, url: games_path do |f| %>
+          <%= f.input :name %>
+          <%= f.input :email %> 
+      </div>
+    </div>
+  </div>
+
   <br />
-  <%= f.submit "Create", class: 'btn btn-primary' %>
-  <% end %>
\ No newline at end of file
+  
+  <div class="choose-army">
+    <div class="row">
+      <div class="col-md-12">
+        <h2>Choose Your Army</h2>
+      </div>
+    <div class="row black-or-white">
+      <div class="white">
+        <%= image_tag ("white_Knight_flip.png") %>
+        <br />
+        <%= f.submit "Play White", class: 'btn white btn-primary' %>
+        <% end %>
+      </div>
+      <div class="black">
+        <%= image_tag ("black_Knight.png") %>
+        <br />
+        <%= simple_form_for @game, url: games_path do |f| %>
+        <%= f.submit "Play Black", class: 'btn black btn-primary' %>
+        <% end %>
```

Comment:
Okay, so it looks like the next step here is to hook this into the logic I'm working on for turns. I should have that pr up soon.